library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/MainActivity.kt

```diff
@@ -150,14 +164,21 @@ internal class MainActivity :
                     getClearDialogData(),
                     onPositiveClick = {
                         viewModel.clearTransactions()
+                        resetSelection()
                     },
                     onNegativeClick = null,
                 )
                 true
             }
             R.id.share_text -> {
                 showDialog(
-                    getExportDialogData(R.string.chucker_export_text_http_confirmation),
+                    getExportDialogData(
+                        if (viewModel.isItemSelected.value == true) {
+                            R.string.chucker_export_text_selected_http_confirmation
+                        } else {
+                            R.string.chucker_export_text_http_confirmation
+                        },
+                    ),
```

Comment:
Because of this, the function is now getting too long and letting detekt fail:

```

> Task :library:detekt
/home/runner/work/chucker/chucker/library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/MainActivity.kt:161:18: The function onOptionsItemSelected is too long (60). The maximum length is 60. [LongMethod]

```

You can see it in the CI signal.

Can you export this + lines 193/199 to separate functions